### PR TITLE
Added Cypress tests for OntoRefine import page

### PR DESCRIPTION
**Pull request by mihailradkov**
Tested that the iframe is properly rendered with its available tabs and panels

### CODE REVIEW

**Comment by mihailradkov** [comment]
This pull request is based on #48 because of the `iframe` Cypress command. When #48 is merged I will rebase this one.

File: test-cypress/integration/import/onto-refine.spec.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    it('should properly render OntoRefine transformation tool frame', () => {
+
+        // Refine should provide navigation tabs
+        getOntoRefineTabs()
+            .should('be.visible')
+            .and('have.length', 4)
+            .each(($tab, $index) => expect($tab.text()).to.equal(REFINE_TABS[$index]))
+            .eq(0)
+            // The first tab should be automatically selected
```
Comment (svilenvelikov) [inline test-cypress/integration/import/onto-refine.spec.js]: and visible maybe?

File: test-cypress/integration/help/rest-api.spec.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
+describe('Help / REST API', () => {
```
Comment (desislava-hristova-ontotext) [inline test-cypress/integration/help/rest-api.spec.js]: Isn't this part of the other merge request for swagger/rest? Seems that should not be here.

Comment (mihailradkov) [reply test-cypress/integration/help/rest-api.spec.js]: Yes, it is. See https://github.com/Ontotext-AD/graphdb-workbench/pull/49#issuecomment-525644596

File: test-cypress/integration/import/onto-refine.spec.js (hunk 1 above)
Comment (mihailradkov) [reply test-cypress/integration/import/onto-refine.spec.js]: Add explicit check that the default tab is visible. The others should be handled by `.click()`'s default checks for visibility.

File: test-cypress/integration/help/rest-api.spec.js (hunk 2 above)
Comment (mihailradkov) [reply test-cypress/integration/help/rest-api.spec.js]: Rebased and skipped that commit.

**Pull request merged**